### ISSUE TITLE
Const metrics defined in lua script not showing up in metrics

### ISSUE THREAD

**Issue by 0019** [oliver006/redis_exporter#546]
I defined a lua script to set a key-value pair:

```
local result={}
table.insert(result, "key1")
table.insert(result, "value1")
return result
```

Then I run the exporter as this:
```
 redis_exporter --redis.addr 10.001.001.46:10001 --script exporter_script.lua
```
Then, I visit the metrics page:
`http://localhost:9121/scrape?target=10.143.140.46:10001`
or 
`http://localhost:9121/metrics`

But my key1 and value1 are not showing up on the page. May I know what is wrong with my usage? Thanks.

**Comment by oliver006** [comment]
`table.insert(result, "value1")` - `value1` is not a valid float so it's not showing up as a metric. If you change it to `table.insert(result, "123")` then it should show just fine.

This could use an error message to better highlight the issue.

**Comment by oliver006** [comment]
Closed by #547

**Comment by 0019** [comment]
Thanks a lot!
